### DISCUSSION TITLE
[Feature IDEA] Automatically Rescan Tags

### DISCUSSION THREAD

**Discussion by trueqap** [usememos/memos#2689] [Ideas]
When a tag arrives from an API, it is inserted as plain text and does not automatically categorize under the appropriate tag. A workaround is to edit and save the memo again. A potential solution is to have the system rescan entries every 5 minutes. Alternatively, the API could recognize hashtags directly.

**Comment by helmut72** [comment]
If Memos supports Front matter for tags like in this feature request #2319 and hashtags are optionally, there is a place for handling tags with the API and you don't get the hashtag `#2319` to your Memos instance, when you just copy/paste my comment as a note into Memos 😉
